ci-operator/jobs/openshift/vertical-pod-autoscaler-operator/openshift-vertical-pod-autoscaler-operator-master-presubmits.yaml

```diff
@@ -44,3 +237,46 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/unit
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: pull-ci-openshift-vertical-pod-autoscaler-operator-master-unit
+    rerun_command: /test unit
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=unit
+        command:
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-vertical-pod-autoscaler-operator-master.yaml
+              name: ci-operator-master-configs
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
```

Comment:
Is name supposed to be empty here?

Reply:
That's the way it generates from the config and I see the same thing in jobs for lots of other projects. I think it should be fine to have a blank container name for the jobs.

Reply:
Resolved via slack.